src/components/ui/map-embed.test.tsx

```
@@ -0,0 +1,229 @@
+    // Update coordinates
+    rerender(<MapEmbed lat={40.7128} lng={-74.0060} />);
+    
+    await waitFor(() => {
+      expect(mockMarker.setLngLat).toHaveBeenCalledWith([-74.0060, 40.7128]);
+    });
+  });
+
+  it('should handle missing Mapbox token gracefully', async () => {
+    // Temporarily remove the mocked token
+    const originalToken = process.env.NEXT_PUBLIC_MAPBOX_ACCESS_TOKEN;
+    delete process.env.NEXT_PUBLIC_MAPBOX_ACCESS_TOKEN;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)